tests/e2e/support/commands.js

```diff
@@ -140,20 +129,20 @@ Cypress.Commands.add('setupGameAsSpectator', () => {
 
     cy.url().should('include', '/spectate/');
     cy.window()
-    .its('cuttle.app.config.globalProperties.$store.state.game')
-    .then((gameState) => {
-      expect(gameState.id).to.not.eq(null);
-    });
+      .its('cuttle.app.config.globalProperties.$store.state.game')
+      .then((gameState) => {
+        expect(gameState.id).to.not.eq(null);
+      });
   });
 });
 
-Cypress.Commands.add('signupOpponent', (username, password) => {
+Cypress.Commands.add('signupOpponent', (opponent) => {
   return new Cypress.Promise((resolve, reject) => {
     io.socket.get(
       'localhost:1337/user/signup',
       {
-        username,
-        password,
+        username: opponent.username,
+        password: opponent.password,
```

Comment:
Then `username` and `password` would already be defined and this part could stay as-is
```suggestion
        username,
        password,
```